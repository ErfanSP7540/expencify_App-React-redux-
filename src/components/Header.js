import React from 'react'
import {NavLink} from 'react-router-dom';

const Header =()=>(
    <div><h1>Header Text</h1>
        <NavLink to="/" exact> Home  </NavLink>
        <NavLink to="/Help" > Help  </NavLink>
        <NavLink to="/Add" > Add </NavLink>
        <NavLink to="/Edit" > Edit  </NavLink>
    </div>
)

export default Header;