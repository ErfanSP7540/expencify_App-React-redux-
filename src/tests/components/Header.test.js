import React from 'react'
import Header from '../../components/Header'

import {shallow} from 'enzyme';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
configure({ adapter: new Adapter() });

test('should renfer header correctly',()=>{

    const wrapper = shallow(<Header />);

    expect( wrapper.find('h1').text() ).toBe('Header Text')
    expect(wrapper).toMatchSnapshot();
    
})

