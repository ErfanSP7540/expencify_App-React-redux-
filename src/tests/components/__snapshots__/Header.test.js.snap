// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should renfer header correctly 1`] = `
<div>
  <h1>
    Header Text
  </h1>
  <NavLink
    activeClassName="active"
    ariaCurrent="true"
    exact={true}
    to="/"
  >
     Home  
  </NavLink>
  <NavLink
    activeClassName="active"
    ariaCurrent="true"
    to="/Help"
  >
     Help  
  </NavLink>
  <NavLink
    activeClassName="active"
    ariaCurrent="true"
    to="/Add"
  >
     Add 
  </NavLink>
  <NavLink
    activeClassName="active"
    ariaCurrent="true"
    to="/Edit"
  >
     Edit  
  </NavLink>
</div>
`;
